tests/utils_tests/test_lazyobject.py

```diff
@@ -284,3 +335,17 @@ def test_list_set(self):
         self.assertNotIn(6, lazy_set)
         self.assertEqual(len(lazy_list), 5)
         self.assertEqual(len(lazy_set), 4)
+
+    # Regression test for #25389
```

Comment:
It might be better to create a separate test class so you can define the new objects right above it instead of having to reference the classes 300 lines above where the test is.